<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowClass</class>
 <widget class="QMainWindow" name="MainWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="mainwindow.qrc">
    <normaloff>:/icon/Resources/sdrcloud.ico</normaloff>:/icon/Resources/sdrcloud.ico</iconset>
  </property>
  <property name="iconSize">
   <size>
    <width>256</width>
    <height>256</height>
   </size>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="actDebug"/>
   </widget>
   <widget class="QMenu" name="menuRTL">
    <property name="title">
     <string>RTL</string>
    </property>
    <addaction name="actOpenRTL"/>
    <addaction name="actCloseRTL"/>
    <addaction name="actDebug"/>
   </widget>
   <widget class="QMenu" name="menuImage">
    <property name="title">
     <string>图像</string>
    </property>
    <addaction name="actOpenImgWindow"/>
    <addaction name="actDebug"/>
   </widget>
   <widget class="QMenu" name="menuSate">
    <property name="title">
     <string>卫星</string>
    </property>
    <addaction name="actDebug"/>
   </widget>
   <widget class="QMenu" name="menuConfig">
    <property name="title">
     <string>设置</string>
    </property>
    <addaction name="actDebug"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="actDebug"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuRTL"/>
   <addaction name="menuImage"/>
   <addaction name="menuSate"/>
   <addaction name="menuConfig"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actDebug">
   <property name="icon">
    <iconset resource="mainwindow.qrc">
     <normaloff>:/icon/Resources/ASX_Cancel_blue_16x.png</normaloff>:/icon/Resources/ASX_Cancel_blue_16x.png</iconset>
   </property>
   <property name="text">
    <string>debug</string>
   </property>
   <property name="toolTip">
    <string>仅用于测试</string>
   </property>
  </action>
  <action name="actOpenRTL">
   <property name="text">
    <string>打开RTL设备...</string>
   </property>
   <property name="toolTip">
    <string>打开一个RTL设备</string>
   </property>
  </action>
  <action name="actCloseRTL">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>关闭RTL设备</string>
   </property>
   <property name="toolTip">
    <string>关闭打开的RTL设备</string>
   </property>
  </action>
  <action name="actOpenImgWindow">
   <property name="text">
    <string>打开图像窗口...</string>
   </property>
   <property name="toolTip">
    <string>打开图像窗口</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="mainwindow.qrc"/>
 </resources>
 <connections/>
</ui>
